src/dish.cpp

```diff
@@ -233,7 +233,6 @@ void zmq::dish_t::send_subscriptions (pipe_t *pipe_)
 
         //  Send it to the pipe.
         pipe_->write (&msg);
-        msg.close ();
```

Comment:
It was actually a mistake to close the msg. Msgs must not be closed after writing to pipe, only re-initialized, it is no the case here as it is used in a loop and returned to the caller.